src/resolvers/ArticleResolver.ts

```diff
@@ -60,11 +60,27 @@ class ArticleResolver {
     return article['_doc'].shoutouts / (presentDate - article['_doc'].date.getTime()); // eslint-disable-line
   }
 
+  @FieldResolver((_returns) => Number)
+  async nsfw(@Root() article: Article): Promise<boolean> {
+    return ArticleRepo.checkProfanity(article['doc'].title); //eslint-disable-line
```

Comment:
I think we could do this in the parsing step. 

We don't store article text, but some RSS feeds do give us this data. This would not only make our NSFW flag more powerful with article content, but it would also prevent having to this categorization in every request.

Reply:
Are you saying the XML in the feed could give us this information?

Reply:
You can use the status code directly, if you receive 304 then you know the feed hasn't been refreshed and can not even parse any of the XML.